aws/provider.go

```diff
@@ -412,6 +412,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_glue_catalog_table":                       resourceAwsGlueCatalogTable(),
 			"aws_glue_classifier":                          resourceAwsGlueClassifier(),
 			"aws_glue_connection":                          resourceAwsGlueConnection(),
+			"aws_glue_catalog_crawler":                     resourceAwsGlueCatalogCrawler(),
```

Comment:
We should probably remove `_catalog` from the resource/file names, and `Catalog` from the function names everywhere to match the API, which just refers to these objects without including catalog in the naming.

You know this already but this resource is missing documentation in `website/docs/r/glue_crawler.html.markdown` and a sidebar link in `website/aws.erb` 😉

Reply:
Done